sql/python/sqlflow_submitter/ant_xgboost/sqlflow_ant_xgboost_main.py

```diff
@@ -69,7 +76,20 @@ def parse_json_str(string: str):
             if not os.path.exists(model_path):
                 os.makedirs(model_path)
             train_fields = cf.TrainFields(learning_fields, data_fields, model_fields)
+            # dump training config
+            config = json.dumps(config_helper.dump_config(train_fields), indent=2)
+            config = re.sub(r'"password": ".*"', '"password": "*****"', config)
+            logging.warning("======xgboost training config======\n%s" % config)
```

Comment:
logginer.info ?

Reply:
Use warning here because info level seems to be ignored by sqlflow streaming.